fhir-smart/src/main/java/com/ibm/fhir/smart/AuthzPolicyEnforcementPersistenceInterceptor.java

```diff
@@ -173,27 +174,67 @@ public void afterInvoke(FHIROperationContext context) throws FHIRPersistenceInte
         return new ArrayList<>(types);
     }
 
+    /**
+     * gets the supported resource types
+     * @return
+     */
+    private List<String> getSupportedResourceTypes() {
+        try {
+            List<String> rts = FHIRConfigHelper.getSupportedResourceTypes();
+            if (!rts.isEmpty()) {
+                return rts;
+            }
+        } catch (FHIRException e) {
+            log.throwing(this.getClass().getName(), "getSupportedResourceTypes", e);
+        }
+        return ModelSupport.getResourceTypes(false)
+                .stream()
+                .map(clz -> clz.getSimpleName())
+                .collect(Collectors.toList());
+    }
+
     private List<String> computeExportResourceTypes(FHIROperationContext context) {
+        List<String> supportedResourceTypes = getSupportedResourceTypes();
         List<String> resourceTypes = new ArrayList<>();
+        Parameters parameters = (Parameters) context.getProperty(FHIROperationContext.PROPNAME_REQUEST_PARAMETERS);
+        Optional<Parameter> typesParam = parameters.getParameter().stream().filter(p -> "_type".equals(p.getName().getValue())).findFirst();
```

Comment:
from https://hl7.org/fhir/uv/bulkdata/export.html#bulk-data-kick-off-request
> A client MAY repeat kick-off parameters that accept comma delimited values multiple times in a kick-off request. The server SHALL treat the values provided as if they were comma delimited values within a single instance of the parameter. Note that we will be soliciting feedback on the use of comma delimited values within parameters, and depending on the response may consider deprecating this input approach in favor of repeating parameters in a future version of this IG.

so we might want to get them all, not just the first one

Reply:
Yep, there is an issue for that #3081